mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,18 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-  return a + b + c;
+function addNumbers(a, b, c) {
+  return a+b+c;
 }
 
 function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
-
-function getTotal(a, b) {
-  total = a ++ b;
-
-  return "The total is total";
+{
+  return "Hello, my name is " + name + "and I am " + age + "years old";
 }
```

Comment:
Hi Ahmed I reviewed your code nice job just there is an error in mandatory 1 the line 14 you need to have spaces between your variables and strings like this :   Expected: "Hello, my name is Sonjide and I am 27 years old"
                                                                          Received: "Hello, my name is Sonjideand I am 27years old"